fixtures/0053-array-of-object-without-values-with-description.md

```diff
@@ -8,41 +8,6 @@
                 <div style="width:100%;height:auto;display:flex;flex-direction:row;flex-wrap:no-wrap;justify-content:flex-start;align-items:stretch;position:relative;border-bottom:none;padding-top:0px;padding-bottom:0px;"><span style="position:absolute;top:8px;right:8px;font-family:&#x27;Source Code Pro&#x27;, monospace;font-size:13px;color:#8A93A3;opacity:0.56;">0</span>
                     <div style="height:auto;width:100%;display:flex;flex-direction:column;flex-wrap:no-wrap;justify-content:flex-start;align-items:flex-start;padding-left:0px;padding-right:0px;">
                         <div style="width:100%;height:auto;display:flex;flex-direction:row;flex-wrap:no-wrap;justify-content:flex-start;align-items:stretch;position:relative;">
-                            <div>
```

Comment:
Mentioned here https://github.com/apiaryio/attributes-kit/pull/445/commits/4d4ec744a8443d8be937e1a4acb5a41df9d7e9f3 - because of https://github.com/apiaryio/drafter/issues/330